hles/transformation/src/test/resources/org/broadinstitute/monster/dap/blank_st_owner_id/records/two_blank_owner_ids_of_three_records.json

```diff
@@ -0,0 +1,1473 @@
+{"record":"87616","redcap_event_name":"baseline_arm_1","field_name":"study_id","value":"87616"}
```

Comment:
Any PII should be redacted out of this fixture data if it hasn't already been.

Reply:
good point, i'll give it a pass